src/main/java/net/dv8tion/jda/internal/entities/UserImpl.java

```diff
@@ -137,7 +137,7 @@ public PrivateChannel getPrivateChannel()
         if (!hasPrivateChannel())
             return null;
         PrivateChannel channel = getJDA().getPrivateChannelById(privateChannel);
-        return channel != null ? channel : new PrivateChannelImpl(privateChannel, this);
+        return channel != null ? channel : new PrivateChannelImpl(getJDA(), privateChannel, this);
```

Comment:
I didn't realize we were doing this. This technically is creating a private channel when we are missing info like `setLastMessageId`. Oh well I guess.

Reply:
Also, we should probably rename `privateChannel` here to `privateChannelId` as the variable isn't fantastically named tbh.